tests/cli/test_buildspec.py

```diff
@@ -150,6 +152,19 @@ def test_buildspec_find_invalid():
     cache.print_invalid_buildspecs(error=False, terse=True, header=True)
 
 
+@pytest.mark.cli
+def test_edit_test():
+    # cache.print
+    buildspecs = []
```

Comment:
No need to declare here u can call it directly in function call `edit_buildspec_test(buildspecs=[], ...